python/ml4ir/applications/ranking/tests/test_fr_one_hot_conversion.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import warnings
+from ml4ir.base.features.preprocessing import convert_fr_to_one_hot
+import numpy as np
+
+warnings.filterwarnings("ignore")
+
+
+class TestFrOneHotConversion(unittest.TestCase):
+
+    def fr_one_hot_conversion(self, fr_value, max_ranks, mask_fr):
```

Comment:
@mohazahran `fr` is internal lingo and you shouldn't use this in `ml4ir`. Why not just use the feature config `rank` attribute instead. It will get you this and was designed for this exact purpose in fact.

Reply:
I didn't know that. I can look into it.